src/style.css

```diff
@@ -1,3 +1,72 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Consider adding a fallback font-family to ensure text is displayed with a default font even if 'Roboto' is not available. For example: `font-family: Roboto, Arial, sans-serif;`.